### DISCUSSION TITLE
Suppress "Ignoring CRS" warnings from geojson vector source?

### DISCUSSION THREAD

**Discussion by NoaMills-USDA** [azavea/raster-vision#2087] [Q&A]
My vector data is all in EPSG:4326, but the geojson files have a "crs" property where the crs is written in a different way. Is it possible for me to suppress the "Ignoring CRS" warning?

**Comment by AdeelH** [comment]
It's currently not possible to suppress it. I agree that it is annoying to see that when that data is already in EPSG:4326. Let me see what can be done about it.

**Comment by AdeelH** [comment]
The warning has been removed in the new release. `GeoJSONVectorSource` can now consume GeoJSONs with arbitrary CRS's. Also, the `ignore_crs_field` is now gone. See [migration guide](https://docs.rastervision.io/en/0.30/migration/v0-21_to_v0-30.html).
